src/core_plugins/kibana/public/visualize/listing/visualize_listing.js

```diff
@@ -1,7 +1,8 @@
 import SavedObjectRegistryProvider from 'ui/saved_objects/saved_object_registry';
 import 'ui/pager_control';
 import 'ui/pager';
-import _ from 'lodash';
+
+import { SortProperties } from 'ui_framework/services';
```

Comment:
I think we're going to find ourselves with a lot of services that are standalone functions, which will all have verb-based names. Since "sort" is also a verb, I think this name is a bit ambiguous. How about renaming it to `SortableProperties`?